### PR TITLE
[Doc] Added license, updated Readme and added Changelog file

### PR DESCRIPTION
**Pull request by Pierstoval**
Ping @impress/mergers for review

### CODE REVIEW

**Comment by Pierstoval** [comment]
@bartaz I made fb0f882 to fix your comments. Could you reply only in the PR to keep the diff/notes up to date to your needs? :)

**Comment by bartaz** [comment]
@Pierstoval sure, I wasn't aware that these comments are not shared across commit and files changed pages. They still appear there in PR thread.

File: README.md [hunk 1]
```diff
@@ -292,12 +136,4 @@ found a good way to handle its small screen.
...
-
-LICENSE
----------
-
-Copyright 2011-2012 Bartek Szopka
-
-Released under the MIT and GPL (version 2 or later) Licenses.
-
-
+Copyright 2011-2016 Bartek Szopka - [Licence](LICENCE)
```
Comment (bartaz) [inline README.md]: Thanks for all the fixes. Still, make clear that license is MIT in readme, so people don't need to look into license file just to know that. So make it `Released under the MIT [License](LICENSE)`. - note the typo in file name.

**Comment by FagnerMartinsBrack** [comment]
The good thing in comment on the "Files Changed" tab is that Github shows a part of the snippet and all comments below it in the timeline

File: README.md (hunk 1 above)
Comment (Pierstoval) [inline README.md]: Fixed in a33f2ec !

**Comment by Pierstoval** [comment]
> The good thing in comment on the "Files Changed" tab is that Github shows a part of the snippet and all comments below it in the timeline

And when a line of code is changed, the note appears as `outdated diff` instead of a global note :wink:

**Pull request merged**